### DISCUSSION TITLE
Winning my first hackathon

### DISCUSSION THREAD

**Discussion by Rishit-dagli** [EddieHubCommunity/support#1747] [Show and tell]
I thought to share my experiences of the first hackathon I won 5 years ago!
https://twitter.com/rishit_dagli/status/1398281991997444099?s=20

**Comment by ashwinexe** [comment]
Amazing to hear that @Rishit-dagli Here's My experience of winning First Prize at a hackathon (P.S today's the day) 
https://twitter.com/ashwin_exe/status/1399066652252114945
